test/test_ops.py

```diff
@@ -345,6 +345,115 @@ def test_out(self, device, dtype, op):
         op(*sample.input, *sample.args, **out_kwargs)
         self.assertEqual(expected, out)
 
+    @ops([op for op in op_db if op.aliases])
+    def test_aliases_consistency_eager(self, device, dtype, op):
+        # Currently, there is a code duplication based on
+        # test_variant_consistency_eager.
+        # To refactor those methods later.
+        samples = op.sample_inputs(device, dtype, requires_grad=True)
+        if len(samples) == 0:
+            self.skipTest("Skipped! No sample inputs!")
+
+        for sample in samples:
+            expected_forward = op(*sample.input, *sample.args, **sample.kwargs)
+            # Computes expected backward
+            # NOTE: backward may fail for some dtypes
+            exception_during_backwards = False
```

Comment:
I agree with the comment above that in a future PR we should look at refactoring and creating a helper function for this code.